pkgs/development/compilers/yosys/plugins/synlig.nix

```
@@ -29,6 +30,15 @@ stdenv.mkDerivation (finalAttrs: {
     fetchSubmodules = false;  # we use all dependencies from nix
   };
 
+  patches = [
+    (fetchpatch {
+      # Fixes https://github.com/chipsalliance/synlig/issues/2299
+      name = "make-compile-for-yosys-0.37.patch";
+      url = "https://github.com/chipsalliance/synlig/pull/2300/commits/3dd46d4769c20b6dd1163310f8e56560b351a211.patch";
```

```suggestion
      url = "https://github.com/chipsalliance/synlig/commit/3dd46d4769c20b6dd1163310f8e56560b351a211.patch";
```
We are getting http 406 on the old url.